### PR TITLE
Enable visual confirmation emails by default [MAILPOET-4931]

### PR DESCRIPTION
**Pull request by lysyjan**
## Description

This PR enables the email editor for confirmation emails by default. The existing installations should use their current configuration.

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-4931]

## After-merge notes

_N/A_

[MAILPOET-4931]: https://mailpoet.atlassian.net/browse/MAILPOET-4931?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/tests/acceptance/Settings/CreateNewWordPressUserCest.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function sendConfirmationEmail(\AcceptanceTester $i) {
     $i->checkEmailWasReceived($emailTitle);
     $i->click(Locator::contains('span.subject', $emailTitle));
     $i->switchToIframe('#preview-html');
-    $i->click('Click here to confirm your subscription.');
+    $i->click('Click here to confirm your subscription');
```
Comment (rodrigoprimo) [inline mailpoet/tests/acceptance/Settings/CreateNewWordPressUserCest.php:45]: Maybe I'm missing something obvious here, but it is not immediately clear to me why this change was needed. For cases like this, I would suggest including in the commit message description an explanation of why removing the dot in this file and in SubscribeOnRegistrationPageCest.php was necessary and what exactly was broken that was fixed.

**Review by rodrigoprimo — APPROVED** [review]
Looks good to me. I left just one comment with a suggestion for future PRs, but that doesn't block this one. Moving it to QA.

**Review by Aschepikov — APPROVED** [review]
GJ!
